pyiceberg/table/__init__.py

```diff
@@ -223,6 +219,9 @@ class TableProperties:
     MIN_SNAPSHOTS_TO_KEEP = "history.expire.min-snapshots-to-keep"
     MIN_SNAPSHOTS_TO_KEEP_DEFAULT = 1
 
+    SNAPSHOT_ID_INHERITANCE_ENABLED = "compatibility.snapshot-id-inheritance.enabled"
+    SNAPSHOT_ID_INHERITANCE_ENABLED_DEFAULT = False
```

Comment:
These seem unused, and I don't think we want to add this to PyIceberg. This was introduced in Java to test V2 features on a V1 table.